store/copr/mpp.go

```diff
@@ -341,13 +341,21 @@ func (m *mppIterator) cancelMppTasks() {
 	}
 
 	// send cancel cmd to all stores where tasks run
+	wg := new(sync.WaitGroup)
 	for addr := range usedStoreAddrs {
-		_, err := m.store.GetTiKVClient().SendRequest(context.Background(), addr, wrappedReq, tikv.ReadTimeoutShort)
-		logutil.BgLogger().Debug("cancel task ", zap.Uint64("query id ", m.startTs), zap.String(" on addr ", addr))
-		if err != nil {
-			logutil.BgLogger().Error("cancel task error: ", zap.Error(err), zap.Uint64(" for query id ", m.startTs), zap.String(" on addr ", addr))
-		}
+		wg.Add(1)
+		go func(storeAddr string) {
+			defer func() {
+				wg.Done()
+			}()
+			_, err := m.store.GetTiKVClient().SendRequest(context.Background(), storeAddr, wrappedReq, tikv.ReadTimeoutShort)
```

Comment:
Do we need to limit the batch number?

Reply:
Well, the max number will be the TiFlash nodes in the cluster. Considering that TiDB always dispatchMPPTask to TiFlash node parallelly without limit, I think here is ok to not limit the batch number